src/screens/RideModeStack/DropOffSubmit.jsx

```diff
@@ -94,6 +97,7 @@ DropOffSubmit.propTypes = {
     navigate: PropTypes.string.isRequired,
     goBack: PropTypes.string.isRequired,
   }).isRequired,
+  route: PropTypes.isRequired,
```

Comment:
What type of object is route? Looks like you want something like:
PropTypes.shape({
  params: PropTypes.shape({
    bike: PropTypes.instanceOf(Bike)
    })
})